scripts/beck/st/error_injection.py

```diff
@@ -200,7 +212,11 @@ def ReadCompleteRecord(offset):
     curr_record = [ hex(int(i, 16)) for i in curr_record]
     return curr_record, offset # Return record data and footer offset
 
+
 def ReadBeBNode(offset):
+    """
+    Read BeNode data
+    """
```

Comment:
Function documentation is in 3rd person, see doc/coding-style.md. That is "Reads", "Induces", etc.